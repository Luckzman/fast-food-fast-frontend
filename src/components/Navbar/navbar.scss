.ui.inverted.segment.no-border-radius {
  border-radius: 0;
  margin: 0 0;
}
