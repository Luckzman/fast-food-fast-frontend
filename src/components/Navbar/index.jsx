import React, { Component } from 'react';
import { Menu, Segment } from 'semantic-ui-react';
import './navbar.scss';

export default class MenuExampleInvertedSegment extends Component {
  state = { activeItem: 'home' };

  handleItemClick = (e, { name }) => {
    this.setState({ activeItem: name });
  };

  render() {
    const { activeItem } = this.state;

    return (
      <Segment inverted className="no-border-radius">
        <Menu inverted secondary>
          <Menu.Item name="Home" active={activeItem === 'Home'} onClick={this.handleItemClick} />
          <Menu.Menu position="right">
            <Menu.Item
              name="catalog"
              active={activeItem === 'catalog'}
              onClick={this.handleItemClick}
            />
            <Menu.Item
              name="login"
              active={activeItem === 'login'}
              onClick={this.handleItemClick}
            />
          </Menu.Menu>
        </Menu>
      </Segment>
    );
  }
}
