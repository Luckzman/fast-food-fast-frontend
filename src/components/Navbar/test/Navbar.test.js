import React from 'react';
import { shallow } from 'enzyme';
import Navbar from '..';

describe('App', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Navbar />);
    expect(wrapper).toMatchSnapshot();
  });
});
