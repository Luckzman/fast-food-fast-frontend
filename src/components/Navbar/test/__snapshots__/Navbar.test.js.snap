// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<Segment
  className="no-border-radius"
  inverted={true}
>
  <Container>
    <Menu
      inverted={true}
      secondary={true}
    >
      <MenuItem
        active={false}
        name="Home"
        onClick={[Function]}
      />
      <MenuMenu
        position="right"
      >
        <MenuItem
          active={false}
          name="catalog"
          onClick={[Function]}
        />
        <MenuItem>
          <ModalContainer />
        </MenuItem>
      </MenuMenu>
    </Menu>
  </Container>
</Segment>
`;
